data/en/text.py

```diff
@@ -249,95 +249,102 @@ def f_RECOVER_NUMBER_HP(power):
-    "basic_halberd": "Basic Halberd",
-    # Effects & Alternations
-    "defense_up": "Defense up",
-    "strength_up": "Strength up",
-    "speed_up": "Speed up",
-    "stun": "Stun",
-    "no_attack": "No attack",
-    # Races
-    "human": "Human",
-    "elf": "Elf",
-    "dwarf": "Dwarf",
-    "centaur": "Centaur",
-    "gnome": "Gnome",
-    # Classes
-    "warrior": "Warrior",
-    "ranger": "Ranger",
-    "spy": "Spy",
-    # Foe keywords
-    "undead": "Undead",
-    "large": "Large",
-    "cavalry": "Cavalry",
-    "mutant": "Mutant",
-    "fly": "Fly",
-    "none": "None",
-    # Entity names
-    "skeleton": "Skeleton",
-    "skeleton_cobra": "Skeleton Cobra",
-    "necrophage": "Necrophage",
-    "lich_boss": "Lich Boss",
-    "mutant_bee": "Mutant Bee",
-    "mutant_lizard": "Mutant Lizard",
-    "mutant_cultist": "Mutant Cultist",
-    "mutant_ant": "Mutant Ant",
-    "obstacle": "Obstacle",
-    "shop": "Shop",
-    "house": "House",
-    "chest": "Chest",
-    "healer": "Healer",
-    "tavern": "Tavern",
-    "door": "Door",
-    "altar": "Altar",
-    "armory": "Armory",
-    "apothecary": "Apothecary",
-    # Attack kinds
-    "physical": "Physical",
-    "spiritual": "Spiritual"
+TRANSLATIONS = {
+    "Items": {
+        "key": "Key",
+        "bones": "Bones",
+        "topaz": "Topaz",
+        "iron_ring": "Iron Ring",
+        "monster_meat": "Monster Meat",
+        "life_potion": "Life Potion",
+        "speed_potion": "Speed Potion",
+        "rabbit_step_potion": "Rabbit Step Potion",
+        "strength_potion": "Strength Potion",
+        "vigor_potion": "Vigor Potion",
+        "scroll_of_knowledge": "Scroll of Knowledge",
+        "scroll_of_cerberus": "Scroll of Cerberus",
+        "chest_key": "Chest Key",
+        "door_key": "Door Key",
+        "green_book": "Green Book",
+        "poket_knife": "Poket Knife",
+        "dagger": "Dagger",
+        "club": "Club",
+        "short_sword": "Short Sword",
+        "wooden_spear": "Wooden Spear",
+        "halberd": "Halberd",
+        "pickaxe": "Pickaxe",
+        "wooden_bow": "Wooden Bow",
+        "basic_bow": "Basic Bow",
+        "wooden_staff": "Wooden Staff",
+        "necromancer_staff": "Necromancer Staff",
+        "plumed_helmet": "Plumed Helmet",
+        "black_hood": "Black Hood",
+        "helmet": "Helmet",
+        "horned_helmet": "Horned Helmet",
+        "gold_helmet": "Gold Helmet",
+        "chainmail": "Chainmail",
+        "leather_armor": "Leather Armor",
+        "scale_mail": "Scale Mail",
+        "gold_armor": "Gold Armor",
+        "spy_outfit": "Spu Outfit",
+        "barding_magenta": "Barding Magenta",
+        "brown_boots": "Brown Boots",
+        "black_boots": "Black Boots",
+        "gold_boots": "Gold Boots",
+        "wooden_shield": "Wooden Shield",
+        "pocket_knife": "Pocket Knife",
+        "basic_spear": "Basic Spear",
+        "basic_halberd": "Basic Halberd",
+    },
+    "Effects&Alternations": {
```

Comment:
I'm sorry but I feel like this naming format is wrong.

It should not be PascalCase, but snake_case.

So `items`, `foe_keywords` etc.

Also, I think effects and alt**era**tions should be splited into different categories, even if they are close. They are close, but should not be confused imo.
Same for races and classes.

Reply:
>It should not be PascalCase, but snake_case.

Okay. Ill rename it.

>Also, I think effects and alterations should be splited into different categories, even if they are close. They are close, but should not be confused imo.

I merged these items because the names seemed to be mixed up in some tests (class was treated as race). Modifying it might introduce some duplication, but I'm at ease with your words. And thanks for pointing out the "alterations" , bruh.

Reply:
I think races and classes should still be merged. because
* races and classes are mixed in some places in the project such as
equipment.py Line104
```python
            if key in ("classes", "races"):
                formatted_string += (
                    ", ".join(
                        [
                            TRANSLATIONS["Races&Classes"][restriction.lower().replace(" ", "_")]
                            for restriction in self.restrictions[key]
                        ]
                    )
                    + ", "
                )
```
Seperating them need new refactor on those mixed parts
* races and classes have a low change of being duplicated

Reply:
I see, yes it is introducing more refactor to split them then let's not split, at least for now.